Directory.Packages.props

```diff
@@ -44,7 +44,7 @@
     <PackageVersion Include="Moq" Version="4.20.70" />
     <PackageVersion Include="Newtonsoft.Json" Version="13.0.3" />
     <PackageVersion Include="NLog.Web.AspNetCore" Version="5.3.8" />
-    <PackageVersion Include="NUnit" Version="4.0.0" />
```

Comment:
Red alert - this changes the package for every outer API - I expect you didn't mean to do that!

Reply:
reverted to original